src/index.js

```diff
@@ -127,6 +127,14 @@ app.setAppUserModelId('com.marshallofsound.gpmdp.core');
 
     // and load the index.html of the app.
     mainWindow.loadURL(`file://${__dirname}/public_html/index.html`);
+
+    // minimize if 'start minimized' is on.
```

Comment:
Thanks for replying! Late is better than never I always say! 😄 I'll be able to take a look at this soon.